cms/envs/devstack.py

```diff
@@ -213,3 +214,12 @@ def should_show_debug_toolbar(request):
 # See if the developer has any local overrides.
 if os.path.isfile(join(dirname(abspath(__file__)), 'private.py')):
     from .private import *  # pylint: disable=import-error,wildcard-import
+
+############# CORS headers for cross-domain requests #################
+FEATURES['ENABLE_CORS_HEADERS'] = True
+CORS_ALLOW_CREDENTIALS = True
+CORS_ORIGIN_WHITELIST = ()
```

Comment:
You don't need to specify a whitelist if `CORS_ORIGIN_ALLOW_ALL == True`.

Reply:
Good point. Let me remove that.